src/features/components/templates/tabs/stepper/index.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
-import Tab from '../../atoms/tab'
+import Tab from '../../../atoms/tab'
```

Comment:
Its better approach to import it like` features/components/atoms/tab`

Reply:
Once we'll move all images to bff and from this place we can't localize it
so, have no sense to spent time on it